src/phoenix/server/api/mutations/api_key_mutations.py

```diff
@@ -53,10 +55,26 @@ class DeleteApiKeyInput:
 
 @strawberry.type
 class DeleteApiKeyMutationPayload:
-    id: GlobalID
+    apiKeyId: GlobalID
     query: Query
 
 
+def can_delete_user_key(info: Info[Context, None], key: OrmApiKey) -> bool:
+    try:
+        user = info.context.request.user  # type: ignore
```

Comment:
why type ignore here?

Reply:
context.request doesn't have a user field on it but it does exist, it looks like, this is what the other code that checks that key is doing